coalib/coala_modes.py

```diff
@@ -61,8 +59,10 @@ def mode_json(args, debug=False):
 
     if args.show_bears:
         try:
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, LogPrinter())
+            from coalib.parsing.FilterHelper import FilterHelper
+
+            local_bears, global_bears = FilterHelper.get_filtered_bears(
+                'language', args.filter_by_language)
```

Comment:
hmm kk. Let's file an issue for that after merge to find some neat idea without duplicating code :+1: